src/ros_answers_miner/cli.py

```diff
@@ -1,25 +1,23 @@
 # -*- coding: utf-8 -*-
 import argparse
 
-from .parser import url_to_question
+from ros_answers_miner.parser import url_to_question
```

Comment:
relative imports are better here; I'd switch back to the original version of the line unless there's a strong reason for the change?